src/main/resources/avro/biodata.avdl

```diff
@@ -0,0 +1,133 @@
+@namespace("org.ga4gh.models")
+
+/**
+This protocol defines the "biodata" objects, which can be considered data
+representations of biological correlates.
+*/
+
+protocol Biodata {
+
+  import idl "common.avdl";
+  import idl "metadata.avdl";
+
+ /**
+  An individual (or subject) typically corresponds to an individual
+  human or other organism.
+  */
+  record Individual {
+
+    /**
+    The Individual's :ref:`id <apidesign_object_ids>`. This is unique in the
+    context of the server instance.
+    */
+    string id;
+
+    /**
+    The Individual's :ref:`name <apidesign_object_names>`. This is a label or
+    symbolic identifier for the individual.
+    */
+    union { null, string } name = null;
+
+    /**
+    The Individual's description. This attribute contains human readable text.
+    The "description" attributes should not contain any structured data.
+    */
+    union { null, string } description = null;
+
+    /**
+    The :ref:`ISO 8601<metadata_date_time> time at which this Individual record
+    was created.
+    */
+    string createDateTime;
+
+    /**
+    The :ref:`ISO 8601<metadata_date_time> time at which this Individual record
+    was updated.
+    */
+    string updateDateTime;
+
+    /**
+    For a representation of an NCBI Taxon ID as an OntologyTerm, see
+    NCBITaxon Ontology
+      http://www.obofoundry.org/ontology/ncbitaxon.html
+    For example, 'Homo sapiens' has the ID 9606. The NCBITaxon ontology ID for
+    this is NCBITaxon:9606, which has the URI
+    http://purl.obolibrary.org/obo/NCBITaxon_9606
+    */
+    union { null, OntologyTerm } species = null;
+
+    /**
+    The genetic sex of this individual.
+    Use `null` when unknown or not applicable.
+    Recommended: PATO http://purl.obolibrary.org/obo/PATO_0020001; PATO_0020002
+    */
+    union { null, OntologyTerm } sex = null;
+
+    /**
+    A map of additional information.
+    */
+    map<array<string>> info = {};
+
+  }
+
+ /**
+  A BioSample refers to a unit of biological material from which the substrate
+  molecules (e.g. genomic DNA, RNA, proteins) for molecular analyses (e.g.
+  sequencing, array hybridisation, mass-spectrometry) are extracted. Examples
+  would be a tissue biopsy, a single cell from a culture for single cell genome
+  sequencing or a protein fraction from a gradient centrifugation.
+  Several instances (e.g. technical replicates) or types of experiments (e.g.
+  genomic array as well as RNA-seq experiments) may refer to the same BioSample.
+  In the context of the GA4GH metadata schema, BioSample constitutes the central
+  reference object.
+ */
+ record BioSample {
```

Comment:
An optional tissue type attribute would be useful. How can tumour and normal samples from the same individual be distinguished?